INSTALL.md

```diff
@@ -281,16 +248,22 @@ packages can be turned off by disabling the relevant options.
 + GINKGO_DEVEL_TOOLS=ON:
   [git-cmake-format](https://github.com/gflegar/git-cmake-format) is our CMake
   helper for code formatting.
++ GINKGO_BUILD_HWLOC=ON:
+  [hwloc](https://www.open-mpi.org/projects/hwloc) to detect and control cores
+  and devices.
 
-By default, Ginkgo uses the internal version of each package. For each of the
+Usually, Ginkgo uses the internal version of each package. For each of the
 packages `GTEST`, `GFLAGS`, `RAPIDJSON` and `CAS`, it is possible to force
 Ginkgo to try to use an external version of a package. For this, Ginkgo provides
 two ways to find packages. To rely on the CMake `find_package` command, use the
-CMake option `-DGINKGO_USE_EXTERNAL_<package>=ON`. Note that, if the external
-packages were not installed to the default location, the CMake option
-`-DCMAKE_PREFIX_PATH=<path-list>` needs to be set to the semicolon (`;`)
-separated list of install paths of these external packages. For more
-Information, see the [CMake documentation for
+CMake option `-DGINKGO_USE_EXTERNAL_<package>=ON`. `HWLOC` works the opposite
+way, Ginkgo always looks for the system's `hwloc` first. Set
+`-DGINKGO_USE_EXTERNAL_HWLOC=OFF` to not use the system provided one.
```

Comment:
does it work?
we set the GINKGO_USE_EXTERNAL_HWLOC=1 when enabling GINKGO_BUILD_HWLOC.
I thought it will override the parameter or have two copies in different scope?

Reply:
@yhmtsai  I have not tried explicitly setting `-DGINKGO_USE_EXTERNAL_HWLOC=OFF`, but without setting this, it does use the system HWLOC at least on Ubuntu 20.04. So at least the default behaviour works correctly.

Reply:
Yes, if a suitable HWLOC is found, then that is **always** used. setting `GINKGO_USE_EXTERNAL_HWLOC=off` makes no difference. 

Do we want the user to be able to force to use the internal version, even if the external version is suitable ? This will probably cause issues, will not have the required functionality and also cause havoc when interfacing with other libraries.

Reply:
I guess I will simply remove this comment then.